hibernate-envers/src/test/java/org/hibernate/envers/test/integration/modifiedflags/HasChangedAPITest.java

```diff
@@ -65,4 +74,18 @@ public void testHasChangedHasNotChangedCriteria() throws Exception {
 		assertTrue( list.isEmpty() );
 	}
 
+	@Test
+	@TestForIssue(jiraKey = "HHH-13770")
+	public void testHasChangedHasNotChangedEnum() {
+		List list = getAuditReader().createQuery().forRevisionsOfEntity( EnumEntity.class, true, true )
+				.add( AuditEntity.property( "option" ).hasChanged() ).getResultList();
+		assertEquals( 2, list.size() );
+		assertEquals( EnumOption.A, ((EnumEntity) list.get( 0 ) ).getOption() );
+		assertEquals( EnumOption.B, ((EnumEntity) list.get( 1 ) ).getOption() );
```

Comment:
It would be perfect to add some empty char between the two `(`s before `EnumEntity`. Super minor, though.

Reply:
Fixed